src/main/java/io/deepstream/rpc/RpcHandler.java

```diff
@@ -55,11 +55,11 @@ public void unprovide( String name ) {
     }
 
     public void make(String name, JsonObject data, RpcResponseCallback callback ) {
-        String uid = this.client.getUid();
-        String typedData = MessageBuilder.typed( data );
+        _make( name, MessageBuilder.typed( data ), callback );

```

Comment:
Yep. My thinking was that we may have 3+ methods for make, ie.

`make( String name, JSONObject data, RpcResponseCallback callback)`
`make( String name, String data, RpcResponseCallback callback)`
`make( String name, int data, RpcResponseCallback callback)`

and extracting the 
`String uid = this.client.getUid();`
`String typedData = MessageBuilder.typed( data );`

from each method and putting it in _make may have made sense